docs/install.md

```diff
@@ -236,10 +236,10 @@ You can specify your own values that replace the default service account (`Servi
 
 The example below customizes the following:
 
-- the default service account from `default` to `tekton`.
-- the default timeout from 60 minutes to 20 minutes.
-- the default `app.kuberrnetes.io/managed-by` label is applied to all Pods created to execute `TaskRuns`.
-- the default Pod template to include a node selector to select the node where the Pod will be scheduled by default. 
+- Service account from `default` to `tekton`.
+- Timeout from 60 minutes to 20 minutes.
```

Comment:
We may want to keep `default` in there in some ways. As it changes the default `timeout` or `serviceAccountName` the `TaskRun` will use if not specified in the `TaskRun` spec.

Reply:
Restored the... defaults.  =)